pages/app/index.tsx

```diff
@@ -72,9 +77,15 @@ function App() {
 
 const history = createHashHistory();
 const { visualRefresh } = parseQuery(history.location.search);
+const { windowVR } = parseQuery(history.location.search);
```

Comment:
Why do we need a new flag? Can we change the logic of the existing query parameter instead?